packages/code-studio/src/settings/AdvancedSectionContent.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useCallback } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  Content,
+  ContextualHelp,
+  Heading,
+  Switch,
+  Text,
+} from '@deephaven/components';
+import {
+  getWebGL,
+  getWebGLEditable,
+  RootState,
+  updateSettings,
+} from '@deephaven/redux';
+
+function AdvancedSectionContent(): JSX.Element {
+  const dispatch = useDispatch();
+  const webgl = useSelector<RootState>(getWebGL) as boolean;
```

Comment:
Not sure why these don't infer the type, but I think `useSelector<RootState, boolean>` will give us better type safety. Same for next one.

Reply:
Actually, looks like this may be a better option ([documented here](https://react-redux.js.org/using-react-redux/usage-with-typescript#define-typed-hooks)):

```typescript
export const useAppSelector: TypedUseSelectorHook<RootState> = useSelector;

...

const webgl = useAppSelector(getWebGL)
```

Reply:
Ohhh nice, much better.